recipes/proteomiqon-peptidespectrummatching/meta.yaml

```diff
@@ -1,25 +1,23 @@
 {% set version = "0.0.7" %}
-{% set sha256 = "720357e04484bf8ea0b5b138798cc676507c10deaf7f3461f994443d466db0d8" %}
+{% set sha256 = "325cdedf33d8edf8a60f4329616fffbd8bd993943e99fbd284a572002138e539" %}
 
 package:
   name: proteomiqon-peptidespectrummatching
   version: '{{ version }}'
 
 source:
-  url: https://www.nuget.org/api/v2/package/ProteomIQon.PeptideSpectrumMatching/{{ version }}
+  url: https://github.com/CSBiology/ProteomIQon/releases/download/peptidespectrummatching-v{{ version }}/ProteomIQon.PeptideSpectrumMatching.v{{ version }}.zip
   sha256: '{{ sha256 }}'
 
 build:
   noarch: generic
-  number: 0
-  script:
-    - dotnet tool install --add-source $SRC_DIR --tool-path $PREFIX/bin --version {{ version }} ProteomIQon.PeptideSpectrumMatching
+  number: 1
 
 requirements:
-  host:
-    - dotnet-sdk
+  build:
   run:
-    - dotnet-runtime
+    - openssl =1.1
```

Comment:
is this pin necessary? It usually not good to bin such a critical library

Reply:
Lets see!

Reply:
https://dev.azure.com/bioconda/bioconda-recipes/_build/results?buildId=15370&view=logs&j=e14e69ff-a0ae-55c4-b71d-229b239cfb2f&t=7df82132-b284-504b-53d6-7d3e63519572&l=1654 Looks like we need it? After all https://github.com/bioconda/bioconda-recipes/blob/master/recipes/flashlfq/meta.yaml probably included it for the same reasons...

Reply:
Aaah, you were asking about the pin... My bad 😊

Reply:
Looks like a hard requirement...